README.md

```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
 _To get help, [submit an issue](https://github.com/jrnl-org/jrnl/issues/new) on
 Github._
 
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
```

Comment:
I think the fact that jrnl uses plain text files is one if its "killer features"--the reason to use this instead of some other program--and it's worth keeping here.